.github/workflows/publish.yaml

```diff
@@ -7,40 +7,38 @@ on:
 
 jobs:
   publish:
-    strategy:
-      matrix:
-        go-version: [1.13.x]
-        os: [ubuntu-latest]
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
-      - name: Install Go
-        uses: actions/setup-go@v2
-        with:
-          go-version: ${{ matrix.go-version }}
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v1
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
       - name: Get TAG
         id: get_tag
         run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+      - name: Set Username/Repo and ImagePrefix as ENV vars
+        run: |
+          echo "USER_REPO=${GITHUB_REPOSITORY,,}" >>${GITHUB_ENV} && \
+          echo "IMAGE_PREFIX=ghcr.io/${GITHUB_REPOSITORY,,}" >>${GITHUB_ENV}
+
       - name: Login to Docker Registry
         uses: docker/login-action@v1
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
           registry: ghcr.io
+          username: ${{ env.USER_REPO }}
```

Comment:
I'm not 100% sure, but I think for me to kick off a release build and to have this work, the value of env.USER_REPO needs to equal my login account name.

The way to test is to create your own org and fork or move your repo there and kick off a build.

Reply:
i tested it in a local fork with my work org, it did work, however, i think we should actually be using `github.repository_owner`

Here you can see the successful login in my fork
![image](https://user-images.githubusercontent.com/891889/132092173-e6639939-9146-41ba-a0dc-7f8be2e39357.png)

And here in the org repo
![image](https://user-images.githubusercontent.com/891889/132092297-6429875f-db93-4e61-8a64-b81c44f0a083.png)